src/main/java/no/nav/pto/veilarbportefolje/auth/AuthService.java

```diff
@@ -58,12 +62,27 @@ public void tilgangTilEnhet(String enhet) {
     }
 
     public boolean harVeilederTilgangTilEnhet(String veilederId, String enhet) {
-        return tryCacheFirst(harVeilederTilgangTilEnhetCache, new VeilederPaEnhet(veilederId, enhet),
+        Boolean abacResponse = tryCacheFirst(harVeilederTilgangTilEnhetCache, new VeilederPaEnhet(veilederId, enhet),
                 () -> veilarbPep.harVeilederTilgangTilEnhet(NavIdent.of(veilederId), EnhetId.of(enhet)));
+
+        if (FeatureToggle.brukPoaoTilgang(unleashService)) {
+            Decision decision = poaoTilgangWrapper.harVeilederTilgangTilEnhet(EnhetId.of(enhet));
+            if (decision.isPermit() != abacResponse) {
```

Comment:
Alexander Gård has actually added this in poao-tilgang:
https://github.com/navikt/poao-tilgang/blob/main/core/src/main/kotlin/no/nav/poao_tilgang/core/policy/impl/NavAnsattTilgangTilNavEnhetPolicyImpl.kt

Reply:
Yep yep, skal fikse..